src/main/java/de/uhd/ifi/se/decision/management/jira/view/diffviewer/BranchDiff.java

```diff
@@ -0,0 +1,157 @@
+package de.uhd.ifi.se.decision.management.jira.view.diffviewer;
+
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElement;
+
+import javax.xml.bind.annotation.XmlElement;
+import java.util.*;
+
+/**
+ * Model class for DiffViewer's branches node
+ */
+public class BranchDiff {
```

Comment:
This class might be better placed in a different package than the view package, e.g. the model package. We should think of adding more model classes for diffs and changed files.

Reply:
Yes, I was also thinking about that, but for the beginning I chose to follow the (no so intuitive) conventions used in the project for treant, treview, vis..

I will refactor this in a new branch, as this problem resolution could be applied to above mentioned classes too.